src/operations/measures/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
"measures" does not seem to be the right word, either "measuring", or "measurements" etc ?

Reply:
measures (noun)
- the dimensions, capacity, or amount of something ascertained by measuring 

measurements (noun)
- a figure, extent, or amount obtained by measuring

I do believe this should be a NOUN. But either is fine with me.

Reply:
After thinking about this for a few days, I favor the shorter 'measures'.

@kaosat-dev What do you think? Is 'measures' fine?

Reply:
This is subjective I guess, but I really am not fond of measures, seems something I would find in politics. Shorter is not really essential :)

Reply:
Done.
I renamed 'mesaures' to 'measurements'.